### DISCUSSION TITLE
Automatically add static release notes in generate_appcast

### DISCUSSION THREAD

**Discussion by vojtabohm** [sparkle-project/Sparkle#2660] [Q&A]
Hey,

I am curious on how to implement release notes so it's as pleasant as possible. I started publishing the release notes at https://steerapp.ai/changelog and I want to embed those on every update. I don't want to tailor custom html files etc. based on a version. Just always load the same change log (which I will update on my website).

However I cannot do it in the `generate_appcast` script. It allows `--release-notes-url-prefix` where I put the URL above but it doesn't add it if you don't have a custom html file for the update as well. Which I do not want. I want each update to have the same releaseNotes link. Right now I manually add it in the xml for each version which is a bit tedious and also, generate_appcast will delete it if I run it again (when creating a new update) which is frustrating.

Any help is appreciated.

Also: fullReleaseNotes is not what is show to the user when the 'a new update is available' window is shown.

**Comment by zorgiepoo** [comment]
Moved to issue #2661
